index.html

```diff
@@ -2863,45 +2890,28 @@ <h3>Add Proof</h3>
         <p>
 The following algorithm specifies how to add a digital proof to a document,
 which  can be used to verify the authenticity and integrity of an
-<dfn>unsecured data document</dfn>. Required inputs are an
-<a>unsecured data document</a> (<var>unsecuredDocument</var>) and
-<a>proof options</a> (<var>options</var>). The
-<a>proof options</a> MUST contain a type identifier for the
-<a>cryptographic suite</a> (<var>type</var>) and any other properties needed by
-the <a>cryptographic suite</a> type; an identifier for the
-<a>verification method</a> (<var>verificationMethod</var>) that can be used to
-verify the authenticity of the proof; an [[!XMLSCHEMA11-2]] `dateTimeStamp`
-string (<var>created</var>) containing the current date and time,
-accurate to at least one second, in Universal Time Code format. A
```

Comment:
I dropped the typestamp because the signing algorithm can retrieve the current date and time just as well as the caller.